tutorial_sarscov2.md

```diff
@@ -0,0 +1,231 @@
+            └── wuhan1_R2.fastq
+```
+
+## Miniconda
+
+V-pipe uses the [Bioconda](https://bioconda.github.io/)[^bioconda] bioinformatics software repository for all component of its pipeline.
+The pipeline itself is written using [snakemake](https://snakemake.readthedocs.io/)[^Snakemake].
+
+[^bioconda]: Grüning, Björn, Ryan Dale, Andreas Sjödin, Brad A. Chapman, Jillian Rowe, Christopher H. Tomkins-Tinch, Renan Valieris, the Bioconda Team, and Johannes Köster. 2018. “Bioconda: Sustainable and Comprehensive Software Distribution for the Life Sciences”. Nature Methods, 2018 doi:[10.1038/s41592-018-0046-7](https://doi.org/10.1038/s41592-018-0046-7).
+
+[^Snakemake]: Johannes Köster and Sven Rahmann. [Snakemake – a scalable bioinformatics workflow engine](https://academic.oup.com/bioinformatics/article/28/19/2520/290322). *Bioinformatics*, 28(19):2520–2522, 2012. doi:[10.1093/bioinformatics/bts480](https://doi.org/10.1093/bioinformatics/bts480)
+
+We're going to set up out environment:
+
+```bash
+# directory for this tutorial
+mkdir V-test
+cd V-test
+
+# Download Miniconda3
+wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+
+# -b for batch (no question asked)
+bash Miniconda3-latest-Linux-x86_64.sh -b -p ~/V-test/miniconda3
+```
+
+And now we're going to start miniconda and install snakemake from bioconda.
+
+```bash
+# mind the dot (=source)
+. ~/V-test/miniconda3/bin/activate
+
+conda create -n V-pipe -c bioconda snakemake conda
+conda activate V-pipe
+# We will let snakemake --use-conda handle
+# installation and download of V-pipe dependencies
+```
+> **Tips:**  To save space, you can install the *-minimal* version of snakemake, without all the GUI dependencies
+
+## Clone V-pipe
+
+
+```bash
+git clone -b sars-cov2 https://github.com/cbg-ethz/V-pipe.git
+cd V-pipe
+```
+> **Versions**: different version come with different defaults
+> - *sars-cov2* branch is adapted for SARS-CoV-2
+> - the default branch (*master*) is adapted for HIV
+
+## Test V-pipe
+
+Put the `samples` hierarchy that you created before in this `V-pipe` directory. Then verify that it has the [desired structure](#preparing-a-small-dataset) :
+
+```bash
+tree samples
+```
+(or alternatively: `find samples`[^fancyfind])
+
+[^fancyfind]: `find samples | sed -e "s/[^-][^\/]*\// |/g" -e "s/|\([^ ]\)/|-\1/"` is overly long but produces slightly prettier output than `find samples`
+
+Check the parameters of the `vpipe.config` file with you favourite editor (`vim`, `emacs`, `nano`, [butterflies](https://xkcd.com/378/), etc.).
+For a SNV and Local (windowed) haplotype reconstruction, you will need at least the following options:
+
+```
+[input]
+reference = references/NC_045512.2.fasta
+
+[output]
+snv = True
+local = True
+global = False
+
+[general]
+aligner = bwa
+```
+
+Check what will be executed:
+```bash
+snakemake -s vpipe.snake --dryrun -p --cores 2
+```
+
+As it is your first run of V-pipe, this will also generate the sample collection table.
```

Comment:
what is the sample-s- collection table and what is it needed for? It is not clear when you follow the tutorial top to bottom.